scripts/travis.sh

```diff
@@ -17,6 +17,8 @@ if [[ "${HEADLESS}" == "true" ]] ; then
     # run SpotBugs only on headless, failing build if bugs are found
     # SpotBugs configuration is in pom.xml
     mvn clean test -U -P travis-spotbugs --batch-mode
+    # run javadoc only on headless
+    mvn javadoc:javadoc  --batch-mode
```

Comment:
I'd remove this line and insert the `javadoc:javadoc` target into the following maven run, so that the line
`mvn test -U -P travis-headless --batch-mode \`
instead reads
`mvn javadoc:javadoc test -U -P travis-headless --batch-mode \`

This avoids the launch overhead running maven twice. Maven can take a list of targets, and will stop on the first failure.